playground/resolve/browser-field/multiple.dot.path.js

```diff
@@ -1,2 +1,2 @@
-const fs = require('node:fs')
+import fs from 'node:fs'
 console.log('this should not run in the browser')
```

Comment:
It looks like this file is intentionally CJS so it doesn't run in the browser, maybe we should leave it as is